tests/unit/test_md_grid.py

```diff
@@ -293,33 +322,91 @@ def test_subdomain_and_interface_iterators(self):
         mdg.add_interface(intf_12, [sd_1, sd_2], None)
         mdg.add_interface(intf_32, [sd_2, sd_3], None)
 
-        # First test traversal by gb.__iter__
+        # First, use the subdomains() function
         found = {sd_1: False, sd_2: False, sd_3: False}
         for sd in mdg.subdomains():
             found[sd] = True
-        self.assertTrue(all([v for v in list(found.values())]))
+        assert all(found.values())
 
-        # Next, use the subdomains() function
-        found = {sd_1: False, sd_2: False, sd_3: False}
-        for sd in mdg.subdomains():
-            found[sd] = True
-        self.assertTrue(all([v for v in list(found.values())]))
-
-        # Finally, check the interfaces
+        # Next, check the interfaces
         found = {intf_12: False, intf_32: False}
         for intf in mdg.interfaces():
             found[intf] = True
-        self.assertTrue(all([v for v in list(found.values())]))
+        assert all(found.values())
+
+        # Finally, check the boundaries
+        parent_grids_found = {sd_1: False, sd_2: False, sd_3: False}
+        for bg in mdg.boundaries():
+            parent_sd = mdg.boundary_grid_to_subdomain(bg)
+            parent_grids_found[parent_sd] = True
+        assert all(parent_grids_found.values())
+
+    def test_iterators_dim_keyword(self):
```

Comment:
Could you add some documentation to this test, please?

Reply:
Fixed